perses/annihilation/new_relative.py

```diff
@@ -0,0 +1,1213 @@
+                                                                            hybrid_index_list[2], angle_parameters[3],
+                                                                            angle_parameters[4])
+
+    def handle_periodic_torsion_force(self):
+        """
+        Handle the torsions in the hybrid system in the same way as the angles and bonds.
+        """
+        old_system_torsion_force = self._old_system_forces['PeriodicTorsionForce']
+        new_system_torsion_force = self._new_system_forces['PeriodicTorsionForce']
+
+        #first, loop through all the torsions in the old system to determine what to do with them. We will only use the
+        #custom torsion force if all atoms are part of "core." Otherwise, they are either unique to one system or never
+        #change.
+        for torsion_index in range(old_system_torsion_force.getNumTorsions()):
+            torsion_parameters = old_system_torsion_force.getTorsionParameters(torsion_index)
+
+            #get the indices in the hybrid system
+            hybrid_index_set = {self._old_to_hybrid_map[old_index] for old_index in torsion_parameters[:4]}
+            #make a list because python set objects can't be indexed
+            hybrid_index_list = list(hybrid_index_set)
+
+            #if all atoms are in the core, we'll need to find the corresponding parameters in the old system and
+            #interpolate
+            if hybrid_index_set.issubset(self._atom_classes['core_atoms']):
+                #get the new indices so we can get the new angle parameters
+                new_indices = [self._topology_proposal.old_to_new_atom_map[old_index] for old_index in torsion_parameters[:4]]
+                new_torsion_parameters = self._find_torsion_parameters(new_system_torsion_force, new_indices)
+
+                #add to the hybrid force:
+                #the parameters at indices 3 and 4 represent theta0 and k, respectively.
+                hybrid_force_parameters = [torsion_parameters[4], torsion_parameters[5], torsion_parameters[6],
+                                           new_torsion_parameters[4], new_torsion_parameters[5],
+                                           new_torsion_parameters[6]]
+                self._hybrid_system_forces['core_torsion_force'].addTorsion(hybrid_index_list[0], hybrid_index_list[1], hybrid_index_list[2], hybrid_index_list[3], hybrid_force_parameters)
+
+            #otherwise, just add the parameters to the regular force:
+            else:
+                self._hybrid_system_forces['standard_torsion_force'].addTorsion(hybrid_index_list[0], hybrid_index_list[1],
+                                                                            hybrid_index_list[2], hybrid_index_list[3], torsion_parameters[4],
+                                                                            torsion_parameters[5], torsion_parameters[6])
+
```

Comment:
Are the torsions of the unique atoms of the new system missing here, or it just doesn't make sense to add them.